### PR TITLE
LC_1639 2-d Dp solution

### PR DESCRIPTION
**Pull request by ajay-mnnit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # daily_dsa
 
-repo for daily coding solutions
+It's a repo for daily coding solutions, we push problems solutions to this repo.
```
Comment (ajay-mnnit) [inline README.md]: please change problems to problem

Comment (karangudha) [inline README.md]: Repo -> repository

File: hard/LC_1639.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Problem: https://leetcode.com/problems/number-of-ways-to-form-a-target-string-given-a-dictionary/description/
+
+//-------------------------------------------------------------------------------------------------------------
+//APPROACH 1: 2-D DP, Ajay Kumar
+// Time: O(n*n)
+// Space: 26*O(n) + O(n*n)
+class Solution {
+    long long mod = 1e9 + 7;
```
Comment (ajay-mnnit) [inline hard/LC_1639.txt]: please write constants in CAPS and make const

File: hard/LC_1639.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                if(i == 0) continue;
+                if(j == 0) 
+                {
+                    dp[i][0] = 1;
+                    continue;
+                }
+                
+                long long cnt = freq[i-1][target[j-1] - 'a'];
+                long long ans = 0;
+
```
Comment (karangudha) [inline hard/LC_1639.txt:36]: Remove this space

**Review by karangudha — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ajay-mnnit) [reply README.md]: sure

File: hard/LC_1639.txt (hunk 3 above)
Comment (ajay-mnnit) [reply hard/LC_1639.txt:36]: done

**Review by karangudha — APPROVED** [review]

**Pull request merged**